core/src/com/unciv/logic/GameInfo.kt

```diff
@@ -286,15 +286,11 @@ class GameInfo {
 
         difficultyObject = ruleSet.difficulties[difficulty]!!
 
-
-
-        // This doesn't HAVE to go here, but why not.
-
-        for (civInfo in civilizations) civInfo.setNationTransient()
-        for (civInfo in civilizations) civInfo.setTransients()
-        for (civInfo in civilizations) civInfo.updateSightAndResources()
-
         for (civInfo in civilizations) {
+            civInfo.setNationTransient()
```

Comment:
Decrease the loop times, so is the below!😄

Reply:
I don't know if this is possible. Maybe `updateSightAndResources()` requires _other_ civilizations to already have their transients set, and if so, this could lead to crashes when loading saves

Reply:
@xlenstra Sorry, you are right!😄
I remove this modification.